pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -459,6 +459,8 @@ message GenerativeSplitAndScatterSpec {
   optional bool exclusive_file_span_comparison = 22 [(gogoproto.nullable) = false];
   // MaxFileCount is the max number of files in an extending restore span entry.
   optional int64 max_file_count = 23[(gogoproto.nullable) = false];
+  // SqlInstanceIds is a slice of SQL instance IDs available for dist restore.
+  repeated int32 sql_instance_ids = 24[(gogoproto.nullable) = false];
```

Comment:
uber nit: specify a custom name so the D is IDs is capitalized. Prior art: https://github.com/msbutler/cockroach/blob/butler-add-workload-gitignore/pkg/sql/execinfrapb/processors_bulk_io.proto#L43